JS/fluentRevealNavbar.uc.js

```diff
@@ -180,6 +237,7 @@
      * @param {boolean} click (whether the left mouse button is down)
      */
     generateEffectsForAll(e, click = false) {
+      this._someEffectsApplied = true;
```

Comment:
You should move this down to the end of the function body, so it doesn't get set if there's an error. Same with the one in `clearEffectsForAll`